document/js-api/index.bs

```diff
@@ -1284,6 +1300,36 @@ The algorithm <dfn>ToWebAssemblyValue</dfn>(|v|, |type|) coerces a JavaScript va
 
 </div>
 
+<div algorithm>
+
+The <dfn method for="Function">type()</dfn> method steps are:
+
+1. Let |store| be the [=surrounding agent=]'s [=associated store=].
+1. Let |funcaddr| be **this**.\[[Function]].
+1. Let |functype| be [=func_type=](|store|, |funcaddr|).
+1. Let |parameters| be |functype|.
+1. Let [|parameters|] → [|results|] be |functype|.
+1. Let |param_types| be « ».
+1. [=list/iterate|For each=] |p| of |parameters|,
+    1. [=list/Append=] [$FromValueType$](|p|) to |param_types|.
+1. Let |result_types| be « ».
+1. [=list/iterate|For each=] |r| of |results|,
+    1. [=list/Append=] [$FromValueType$](|r|) to |result_types|.
+1. Return «[ "{{FunctionType/parameters}}" → |param_types|, "{{FunctionType/results}}" → |result_types| ]».
+
+</div>
+
+<div algorithm>
+  To <dfn>construct a new WebAssembly Function</dfn> from a JavaScript [=callable=] object |callable| and {{FunctionType}} |signature|, perform the following steps:
+
+    1. Assert: [=IsCallable=](|callable|).
+    1. Let |store| be the [=surrounding agent=]'s [=associated store=].
+    1. Let |signature| be «[ "{{FunctionType/parameters}}" → |parameters|, "{{FunctionType/results}}" → |results| ]».
+    1. Let [|parameters|] → [|results|] be |functype|.
```

Comment:
Don't you have to translate parameters and returns first with `ToValueType`?

Reply:
Nice catch! Thanks